lib/wed/object_check.js

```diff
@@ -68,7 +68,7 @@ function _required(template, name) {
  *
  * @param {Object} template The template to use for the check.
  * @param {Object} object The object to check
- * @returns {Object.<string, Array.<string>>}} The results.
+ * @returns {Object.<string>, Array.<string>} The results.
```

Comment:
Yes, I misread that. I have amended that commit to reinstate that line to the original.